### ISSUE TITLE
Suche nach CIP-Pool-Namen ermöglichen

### ISSUE THREAD

**Issue by FloEdelmann** [lmu-navigator/android#21]
Häufig wird bei einer Veranstaltung nur angegeben, dass sie z.B. im *CIP-Pool Sibirien* stattfindet. Nach diesem kann ich aber in der App nicht suchen, nur nach der Raumnummer. Also muss ich diese erst auf der IFI-Seite (https://www.ifi.lmu.de/studium/rechner-pools) nachschlagen. Es wäre schön, wenn zusätzliche Namen eines Raumes gespeichert und durchsuchbar wären.

**Comment by MaxMediaPictures** [comment]
Das wäre eigentlich auch für weitere Räume interessant. Zum Beispiel "Große Aula", "Audimax", ...
Wenn ich mich richtig erinnere hatten wir mal eine Raumbenennung angedacht, aber nie umgesetzt. @dmo60 wir könnten im Backend ein weiteres Feld für Raumnamen hinzufügen. Stellt sich nur die Frage wie bei Updates behandelt wird.

**Comment by FloEdelmann** [comment]
Vielleicht sollten die Raumnamen so wie die Fotos von den Gebäuden behandelt werden: Keine „offizielle“ Information, sondern von den Usern ergänzte Daten.

**Comment by timoloewe** [comment]
Klar, die Idee ist gut und macht Sinn. Ich sehe nur das Problem, dass wir diese Informationen zwar für ein CIP-Räume hinzufügen können, aber viel mehr auch nicht (mal abgesehen von Audimax, etc.). Wenn das für alle nützlich sein soll, und nicht nur für Informatiker, dann bräuchten wir die gleichen Daten von allen möglichen Leuten/Studenten/Fächern/Gebäuden. Wie machen wir das? Ansonsten würden wir ein Feature hinzufügen, dass nur ein paar Informatiker zu sehen bekommen, das fände ich etwas schade.
Das zweite Problem hast du angesprochen @FloEdelmann: Wir würden dann inoffizielle Daten mit den offiziellen Daten der Uni mischen. Ich weiß nicht, ob das ein Problem wäre, aber wir müssten bestimmt mal nachfragen.

**Comment by FloEdelmann** [comment]
Spontan fallen mir noch ein paar andere benannte Räume ein:

* Großer Physikhörsaal (Geschwister-Scholl-Platz 1, N 120)
* Kleiner Physikhörsaal (Geschwister-Scholl-Platz 1, N 020)
* Stucafé (Geschwister-Scholl-Platz 1, C U117)
* Café Gumbel (Theresienstr. 37/39/41, B 029)

Da haben dann auch schon Physiker, Mathematiker und alle, die essen wollen, was davon :)

Natürlich merkt man sich die Räume zu den Namen recht bald, sodass man nicht mehr nachschauen muss, aber für manche ist es bestimmt trotzdem praktisch.

**Comment by timoloewe** [comment]
Ich habe es auf jeden Fall mal auf meiner Liste. Danke, @FloEdelmann! Falls du Interesse hast, selbst bei der Umsetzung zu helfen, ist das natürlich auch willkommen!
Ansonsten kann ich bzw. wir im Moment nichts versprechen, wann und ob wir das in näherer Zukunft realisieren können.
